README.md

```diff
@@ -266,6 +266,32 @@ registry:
 kubeclarity scan registry/nginx:private --config $HOME/own-kubeclarity-config
 ```
 
+### Analyzer output formats
+
+The kubeclarity-cli analyze command can format the resulting SBOM into
+different formats if required to integrate with another system. The supported
+formats are:
+
+| Format | Configuration Name |
+| --- | --- |
+| CycloneDX JSON (default) | cyclonedx-json |
+| CycloneDX XML | cyclonedx-xml |
+| SPDX JSON | spdx-json |
+| SPDX Tag Value | spdx-tv |
+| Syft JSON | syft-json |
+
+
+> ***WARNING***  
+> kubeclarity processes CycloneDX internally, the other formats are supported
+> through a conversion so may not support all the same fields/information.
```

Comment:
Maybe:
... through a conversion. The conversion process can be lossy, so not all fields/information are promised to be present in the result output.